addon/authenticators/jwt.js

```diff
@@ -120,11 +122,11 @@ export default TokenAuthenticator.extend({
 
```

Comment:
@alechirsch I've updated this to attempt a refresh if the JWT is expired. If the server rejects the request the session is invalidated.